src/modules/dashboard/components/ActionsPage/CountDownTimer/CountDownTimer.tsx

```diff
@@ -143,6 +143,12 @@ const CountDownTimer = ({
     walletAddress,
   ]);
 
+  useEffect(() => {
+    if (state && data) {
+      refetch();
+    }
+  }, [state]);
```

Comment:
This needs to have both `data` and `refetch` as dependencies! Please don't ignore `eslint` rules